Update the basic access  permissions to audit (#4475)
MicrosoftDocs/windows-itpro-docs: windows/security/threat-protection/auditing/apply-a-basic-audit-policy-on-a-file-or-folder.md

    -   To audit successful events, click **Success.**
    -   To audit failure events, click **Fail.**
    -   To audit all events, click **All.**

> 6.  In the **Applies to** box, indicate to which object or objects the audit of events will apply, can be to:

Reviewer: ```suggestion
6.  In the **Applies to** box, click the object or objects that the audit of events will apply to. It can be to:
```

    -   **This folder only.**
    -   **This folder, subfolders and files.**
    -   **This folder and subfolders.**
    -   **This folder and files.**
    -   **Subfolders and files only.**

>     -   **Subfolders only**

Reviewer: ```suggestion
    -   **Subfolders only.**
```

    -   **Files only.**

> 7.  By default the selected **Basic Permissions** to Audit are the following:

Reviewer: ```suggestion
7.  By default, the selected **Basic Permissions** to Audit are the following:
```

    -   **Read & Execute.**
    -   **List folder contents.**
    -   **Read.**

>     -   You can additionally select the audit of **Full control**, **Modify** and/or **Write** permissions.  With your desired combination.

Reviewer: ```suggestion
    -   Additionally, you can choose **Full control**, **Modify**, and/or **Write** permissions with your selected audit combination.
```
